pkg/fission-cli/cmd/spec/apply.go

```diff
@@ -117,11 +117,11 @@ func (opts *ApplySubCommand) run(input cli.Input) error {
 		}
 
 		// validate
-		err = fr.Validate(input)
+		warning, err := fr.Validate(input)
 		if err != nil {
 			return errors.Wrap(err, "error validating specs")
 		}
-
+		_ = warning
```

Comment:
Hi @anubhav6663 
if we have to swallow the warning we can use `_` at the line 120 itself. But please make sure Validate prints the warnings.

Reply:
Yeah `Validate` prints the warnings.

Reply:
then I suggest to remove line `124` and have it in `120`